### DISCUSSION TITLE
Where can I find source code for the now removed `servers`

### DISCUSSION THREAD

**Discussion by morgsmccauley** [WalletConnect/walletconnect-monorepo#1198] [Developer Support]
The `servers/` directory was removed in https://github.com/WalletConnect/walletconnect-monorepo/commit/59f4cbad and I am unable to find where they have been moved to. [ops/js/README.md](https://github.com/WalletConnect/walletconnect-monorepo/blob/v2.0/ops/js/README.md?plain=1#L7) mentions a separate repository for the relay server which seems to be private. Where can I find the latest source code?

**Comment by rollycodes** [comment]
Bumpy bump please!

I am trying to setup a bridge/relay server, but:

1. Repo head has server code removed
2. Versions till 2.0.0-beta.101 have it but "make deploy as per README" doesn't work because Makefile doesn't now have a deploy operation
3. Versions till 2.0.0-beta.26 have a Makefile as per README but the "make deploy" errors out there because of missing caddy.
4. Older versions with caddy issue not present fail the "make deploy" because of some or the other git repo dependency not existing anymore!

I have spent atleast 48 hrs trying to test every single possible version out there but failed! Any help here will be highly appreciated!

**Reply by tfalencar** [reply]
@rollycodes did you succeed in the end?
